explorer/interpreter/interpreter.cpp

```diff
@@ -1503,6 +1503,72 @@ auto Interpreter::StepStmt() -> ErrorOr<Success> {
         }
       }
     }
+    case StatementKind::For: {
+      static int CURRENT_INDEX_POS = 1;
+      static int TARGET_VAR_POS = 0;
+      static int END_INDEX_POS = 2;
+      if (act.pos() == 0) {
+        return todo_.Spawn(
+            std::make_unique<ExpressionAction>(&cast<For>(stmt).loop_target()));
+      }
+      if (act.pos() == 1) {
+        Nonnull<const TupleValue*> source_array =
+            cast<const TupleValue>(act.results().back());
+
+        auto end_index = static_cast<int>(source_array->elements().size());
+        auto start_index = 0;
```

Comment:
I think it would be clearer to delete this variable and use a literal `0` instead.